cmake/FindGLIB.cmake

```diff
@@ -0,0 +1,27 @@
+function(FindGLIB)
```

Comment:
This breaks Conan builds. Since functions have its own scope in CMake and `find_package` doesn't promote its variables to the global scope by default it should be refactored to either work on global scope, become a macros (again...) or the logic in the `NOT glib_FOUND` should be rewritten since now it is always executed, even when `glib` package is found by `find_package` (which is probably another side-effect of using a function vs a macro)